schemas/context/experienceevent.schema.json

```diff
@@ -88,6 +88,12 @@
           "description":
             "The information related to web page and link of the ExperienceEvent."
         },
+        "xdm:directMarketing": {
+          "title": "Direct Marketing",
+          "$ref": "https://ns.adobe.com/xdm/context/direct-marketing",
+          "description":
+            "The events and properties related to direct/outbound marketing such as email, direct mail, texts and in-app notifications."
+        },
```

Comment:
Can you also change the examples at https://github.com/adobe/xdm/tree/master/extensions/adobe/experience/campaign with this change?

Reply:
done